### PR TITLE
Add Paula Stepinska user

### PR DESCRIPTION
**Pull request by mybits**

### CODE REVIEW

**Comment by surminus** [comment]
Can you try repushing? It hasn't contacted Jenkins for some reason.

**Comment by h-lame** [comment]
I think that's 'cos @mybits hasn't been added to a team that has write access to the repo (yet).  I'll push up a copy of her branch to our repo to trigger the build.

File: modules/users/manifests/paulastepinska.pp [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Creates the paulastepinska user
+class users::paulastepinska {
+  govuk_user { 'paulastepinska':
+    fullname => 'Paula Stepinska',
+    email    => 'paulas.tepinska@digital.cabinet-office.gov.uk',
```
Comment (h-lame) [inline modules/users/manifests/paulastepinska.pp]: The `.` looks like it's in the wrong place here.

Comment (mybits) [inline modules/users/manifests/paulastepinska.pp]: Oh-oh fixed!

**Comment by surminus** [comment]
Jenkins is still sat there 🤔

**Comment by h-lame** [comment]
I repushed the branch.  Jenkins is doing it's thing now.

**Pull request merged**